<launch>
    <arg name="robot_name" default="lolo"/>

    <push-ros-namespace namespace="$(var robot_name)"/>
    <node pkg="serial_parser" exec="serial_parser">
        <param name="port" value="/dev/ttyUSB1"/>
        <param name="baudrate" value="9600"/>
        <param name="topic_name" value="sensors/dissolved_oxygen/raw"/> <!-- Topic to publish the parsed data -->
        <param name="poll_rate" value="10"/> <!-- Poll rate in Hz -->

        <!-- Command to send to the serial device -->
        <!-- If value="", then the node sends nothing and just streams data from serial port -->
        <param name="send_command" value="stdo,19\r\n"/>

    </node>

</launch>